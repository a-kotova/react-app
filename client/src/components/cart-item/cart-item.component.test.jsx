import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import CartItem from './cart-item.component';

describe('Cart Item Component', () => {
  it('should match snapshot', () => {
    const item = {
      imageUrl: 'somepath',
      name: 'name',
      price: 100,
      quantity: 15,
    };

    const wrapper = shallow(<CartItem item={item} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
