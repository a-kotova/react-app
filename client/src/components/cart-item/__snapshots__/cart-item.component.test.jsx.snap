// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart Item Component should match snapshot 1`] = `
<styled.div>
  <img
    alt="item"
    src="somepath"
  />
  <styled.div>
    <styled.span>
      name
    </styled.span>
    <styled.span>
      15x100
    </styled.span>
  </styled.div>
</styled.div>
`;
